mungegithub/Dockerfile-shame-mailer

```diff
@@ -12,36 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The following environment variable need to be bound before executing
-# this containter:
-#
-#   GITHUB_API_TOKEN: An API token to use with Github's API.
```

Comment:
You're right. I'm deleting the shame mailer code in #3470 though so ill just leave these files alone in this PR.